src/snowflake/snowpark/table_function.py

```diff
@@ -106,6 +108,33 @@ def over(
             new_table_function._order_by = order_spec
         return new_table_function
 
+    def as_(self, *aliases: str) -> "TableFunctionCall":
```

Comment:
actually you can just do `as_ = alias` (we should also do that in column.py actually 🤦 )